org.testeditor.web.dropwizard/src/test/java/org/testeditor/web/dropwizard/ExampleApplication.xtend

```diff
@@ -11,18 +11,33 @@ import org.testeditor.web.dropwizard.auth.User
 
 import static javax.ws.rs.core.Response.Status.*
 import static javax.ws.rs.core.Response.status
+import javax.annotation.security.PermitAll
 
 class ExampleApplication extends DropwizardApplication<ExampleConfiguration> {
 
     override run(ExampleConfiguration configuration, Environment environment) throws Exception {
         super.run(configuration, environment)
         environment.jersey => [
             register(HelloWorldResource)
+            register(HelloUniverseResource)
             register(UserResource)
         ]
     }
 
+	// no authorization restriction
+    @Path("/hellouniverse")
+    @Produces(MediaType.TEXT_PLAIN)
+    public static class HelloUniverseResource {
```

Comment:
Maybe these test resources could be named wrt their intended use, e.g. `NoAuthorizationRestrictionResource`.
Makes the comment obsolete, and its easier to understand the corresponding test cases.

Reply:
Or `AccessibleWithoutAuthorizationResource`